### PR TITLE
Add Robust OvR TPR metric

### PR DESCRIPTION
**Pull request by tmke8**
And don't report diffs and ratios if there are too many y and s values.

### CODE REVIEW

File: src/evaluation/metrics.py [hunk 1]
```diff
@@ -93,28 +94,38 @@ def compute_metrics(
...
+    y_pred_t = torch.as_tensor(torch.as_tensor(predictions.hard, dtype=torch.long))
+    y_true_t = torch.as_tensor(torch.as_tensor(actual.y, dtype=torch.long))
+    s_t = torch.as_tensor(torch.as_tensor(actual.s, dtype=torch.long))
+
+    # compute EthicML metrics
     predictions._info = {}  # type: ignore
     metrics = emm.run_metrics(
         predictions=predictions,
         actual=actual,
         metrics=[emm.Accuracy(), emm.TPR(), emm.TNR(), emm.RenyiCorrelation()],
```
Comment (MylesBartlett) [inline src/evaluation/metrics.py:119]: We could do away with the EM-based metrics altogether, tbh; this would greatly simplify things

Comment (tmke8) [reply src/evaluation/metrics.py:119]: does conduit have everything we need?

**Comment by tmke8** [comment]
I agree that we should align the metrics computations with hyaline at some point, but I'll just merge this for now.

File: src/evaluation/metrics.py (hunk 1 above)
Comment (MylesBartlett) [reply src/evaluation/metrics.py:119]: Should do; it doesn't have Renyi Correlation we only have that here because 'why not?'

**Pull request merged**